plugins/com.google.cloud.tools.eclipse.preferences/src/com/google/cloud/tools/eclipse/preferences/AnalyticsOptInFieldEditor.java

```diff
@@ -0,0 +1,73 @@
+package com.google.cloud.tools.eclipse.preferences;
+
+import org.eclipse.jface.preference.BooleanFieldEditor;
+import org.eclipse.jface.preference.FieldEditor;
+import org.eclipse.jface.preference.IPreferenceStore;
+import org.eclipse.swt.SWT;
+import org.eclipse.swt.layout.GridData;
+import org.eclipse.swt.widgets.Composite;
+import org.eclipse.swt.widgets.Group;
+import org.eclipse.swt.widgets.Link;
+
+public class AnalyticsOptInFieldEditor extends FieldEditor {
+
+  private Group group;
+  private BooleanFieldEditor optInStatusEditor;
+
+  /**
+   * @param name the name of the preference this field editor works on
+   * @param parent the parent of the field editor's control
+   */
+  public AnalyticsOptInFieldEditor(String name, Composite parent) {
+    group = new Group(parent, SWT.SHADOW_OUT);
+    group.setText(Messages.FIELD_EDITOR_ANALYTICS_GROUP_TITLE);
+
+    optInStatusEditor =
+        new BooleanFieldEditor(name, Messages.FIELD_EDITOR_ANALYTICS_OPT_IN_TEXT, group);
+
+    Link link = new Link(group, SWT.NONE);
+    link.setText(Messages.FIELD_EDITOR_ANALYTICS_DISCLAIMER);
```

Comment:
You need to add a selection listener to the `Link` instance.  `event.text` will be the `href` text.  To open a browser, using the platform browser support (which allows opening an internal browser):
```
IWorkbenchBrowserSupport browserSupport = PlatformUI.getWorkbench().getBrowserSupport();
String browserId = null;
IWebBrowser browser = browserSupport.createBrowser(browserId);
browser.openURL(url);
```